features/configuration/custom_settings.feature

```diff
@@ -14,18 +14,14 @@ Feature: custom settings
           expect(RSpec.configuration.custom_setting).to be_nil
         end
 
-        it "acts false by default" do
-          expect(RSpec.configuration.custom_setting).to be_falsey
-        end
```

Comment:
This is part of the documentation, not a copy of another spec so should be restored.

Reply:
I removed it thinking that:
1) `be_falsey` should not be encouraged
2) it is completely redundant with the previous spec (impossible to pass this and not pass the previous spec)
3) it does not document in any way `custom_setting`, only that we can always `expect(nil).to be_falsey`